client-v2/src/shared/components/__tests__/DragIntentContainer.spec.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { render, cleanup, fireEvent } from 'react-testing-library';
+import DragIntentContainer from '../DragIntentContainer';
+
+afterEach(cleanup);
+jest.useFakeTimers();
+
+describe('DragIntentContainer', () => {
+  it('should fire onIntentConfirm callback 300ms after Drag has entered', async () => {
+    const callback1 = jest.fn();
```

Comment:
minor: could this have a more descriptive name, just to make it obvious what's happening further down the test?